app/src/main/res/values-es/strings.xml

```diff
@@ -46,6 +46,7 @@
     <string name="locations">Ubicaciones</string>
     <string name="favorites">Favoritos</string>
     <string name="settings">Configuracion</string>
+    <string name="tour_points">Puntos Turisticos</string>
```

Comment:
The name for the bottom_nav might be too big when in spanish

Reply:
i change it for "Sitios" or maybe should it be "Lugares"